core/src/main/java/com/linecorp/armeria/common/StringValueConverter.java

```diff
@@ -128,6 +128,12 @@ public String convertFloat(float value) {
 
     @Override
     public boolean convertToBoolean(String value) {
+        if ("True".equals(value)) {
+            return true;
+        }
+        if ("False".equals(value)) {
+            return false;
+        }
         return Boolean.parseBoolean(value);
```

Comment:
I found out that this logic is incompatible with how annotated services handle boolean values. 
Should we 1) add `True` and `False` to the `StringUtil.stringToBoolean` map and 2) replace `Boolean.parseBoolean` with `StringUtil.toBoolean()`